extensions/control-plane/api/management-api/contract-negotiation-api/src/test/java/org/eclipse/edc/connector/api/management/contractnegotiation/transform/ContractNegotiationToContractNegotiationDtoTransformerTest.java

```diff
@@ -63,6 +69,7 @@ void transform() {
         assertThat(dto.getErrorDetail()).isEqualTo("errorDetail");
         assertThat(dto.getUpdatedAt()).isEqualTo(contractNegotiation.getUpdatedAt());
         assertThat(dto.getCreatedAt()).isEqualTo(contractNegotiation.getCreatedAt());
+        assertThat(dto.getCallbackAddresses()).usingRecursiveFieldByFieldElementComparator().contains(callback);
```

Comment:
nit: may want to verify that it contains _only_ the `callback` object.

Reply:
Changed the assertion after introducing the `CallbackAddressDto`